src/containers/Profile/ProfileDetails.js

```diff
@@ -32,11 +74,40 @@ const ProfileDetailsComponent = props => {
     toggleNetworkSwitch,
     currentTab,
     setActiveTab,
-    setReviewsModalVisible
+    setReviewsModalVisible,
+    onEdit,
+    onCloseProgressBar,
+    showBanner
   } = props;
 
   let bodyClass;
 
+  const percentageComplete = getProfileCompletePercent(user);
+
+  const pageBanner = (
+    <React.Fragment>
+      <PageBanner>
+        <span style={{ width: '50px' }} />
```

Comment:
a lot of these spans will need to go away or turn to divs once this is switched to flexbox